libobs-opengl/gl-nix.c

```diff
@@ -31,7 +30,7 @@ static void init_winsys(void)
 
 	switch (obs_get_nix_platform()) {
 	case OBS_NIX_PLATFORM_X11_GLX:
-		gl_vtable = gl_x11_glx_get_winsys_vtable();
+		exit(1);
```

Comment:
```suggestion
```
How about just removing the case item `OBS_NIX_PLATFORM_X11_GLX` or put `break` so that it will reach to `assert` at the end?

Reply:
Sorry, I had removed this locally but didn't push afterwards... this is gone now.